internal/context.go

```diff
@@ -284,3 +322,90 @@ func createContextFileIfNotExist() {
 		log.Verboseln("Created ~/.corectl/contexts.yml for storage of corectl contexts")
 	}
 }
+
+func getSessionCookie(tlsClientConfig *tls.Config, engineURL string, userName string, password string) string {
+	// Verify Qlik Sense URL
+	u, err := url.Parse(engineURL)
+
+	if err != nil {
+		log.Fatalln("The engineURL doesn't seem to be correct")
+	}
+
+	if u.Scheme != "https" {
```

Comment:
Do we allow `wss` in the url or will that be handled earlier?

Reply:
Not for login since that has to be done through the proxy but it´s valid for the context soo can´t be checked earlier, or?

Reply:
You can use the `internal/urlparser.go` and use the `parseEngineURL` function. Otherwise we will handle this specific url differently.
The result URL will have ws/wss as scheme which you can just set to https.